app/controllers/my_module_repositories_controller.rb

```diff
@@ -28,6 +33,69 @@ def index_dt
     render 'repository_rows/index.json'
   end
 
+  def update
+    if params[:rows_to_assign]
+      assign_service = RepositoryRows::MyModuleAssigningService.call(my_module: @my_module,
```

Comment:
Didn't we agree that we use single service for assigning/unassigning?

Reply:
We decide to make it in separate task